pt_PT.ini

```diff
@@ -183,48 +183,48 @@ VFPU = VFPU
 
 [Dialog]
 Back = Voltar
-Cancel = Cancel
-ChangingGPUBackends = Changing GPU backends requires PPSSPP to restart. Restart now?
-Choose PPSSPP save folder = Choose PPSSPP save folder
-Confirm Overwrite = Do you want to overwrite the data?
-Confirm Save = Do you want to save this data?
-ConfirmLoad = Load this data?
+Cancel = Cancelar
+ChangingGPUBackends = Mudar de motor de GPU requer que o PPSSPP reinicie. Reiniciar agora?
+Choose PPSSPP save folder = Escolher pasta de gravação do PPSSPP
+Confirm Overwrite = Deseja sobreescrever estes dados?
+Confirm Save = Deseja gravar estes dados?
+ConfirmLoad = Carregar estes dados?
 Delete = Apagar
-Delete all = Delete all
-Delete completed = Excluindo com sucesso!
-DeleteConfirm = This save data will be deleted.\nAre you sure you want to continue?
-DeleteConfirmAll = Do you really want to delete all\nyour save data for this game?
-DeleteConfirmGame = Do you really want to delete this game\nfrom your device? You can't undo this.
-DeleteConfirmGameConfig = Do you really want to delete the settings for this game?
-DeleteFailed = Unable to delete data.
+Delete all = Apagar tudo
+Delete completed = Apagado com sucesso!
+DeleteConfirm = Estes dados de gravação serão apagados. Tem a certeza que quer continuar?
+DeleteConfirmAll = Tem a certeza que deseja apagar\ntodos os seus dados gravados deste jogo?
+DeleteConfirmGame = Tem a certeza que deseja apagar\neste jogo do dispositivo? Isto não poderá ser desfeito.
+DeleteConfirmGameConfig = Tem a certeza que deseja apagar as configurações para este jogo?
+DeleteFailed = Erro ao apagar dados.
 Deleting = Apagando, Aguarde...
 Enter = Aceitar
 Finish = Terminar
-Load = Load
+Load = Carregar
 Load completed = Carregado com sucesso!
 Loading = Carregando
-LoadingFailed = Unable to load data.
-Move = Move
-Network Connection = Network Connection
-NEW DATA = NEW DATA
+LoadingFailed = Erro ao carregar dados.
+Move = Mover
+Network Connection = Ligação de rede
+NEW DATA = Novos dados
 No = Não
 OK = OK
-Old savedata detected = Old savedata detected
-Reset = Reset
-Resize = Resize
-Retry = Retry
-Save = Save
-Save completed = Salvar concluída
-Saving = Salvando Jogo, Aguarde...
-SavingFailed = Unable to save data.
+Old savedata detected = Dados gravados antigos detectados
+Reset = Reiniciar
+Resize = Redimensionar
+Retry = Tentar novamente
+Save = Gravar
+Save completed = Gravação concluída
+Saving = Gravando jogo, aguarde...
```

Comment:
The use of the gerund is privileged in the Brazilian and African variants of Portuguese. I would stick with the infinitive preceded by a preposition (e.g. `A gravar o jogo. Aguarde...`).